prefect_azure/__init__.py

```diff
@@ -3,6 +3,10 @@
     AzureBlobStorageCredentials,
     AzureCosmosDbCredentials,
     AzureMlCredentials,
+    ACICredentials
 )
+from .aci import ACITask
```

Comment:
```suggestion
```
The precedent thus far is only exposing credentials at top level, unless the collection is small.